Source/rclUE/Public/Srvs/ROS2SpawnEntitiesSrv.h

```diff
@@ -0,0 +1,254 @@
+// Copyright 2021 Rapyuta Robotics Co., Ltd.
+// This code has been autogenerated from UE_msgs/SpawnEntities.srv - do not modify
+
+#pragma once
+
+#include <CoreMinimal.h>
+
+#include "Srvs/ROS2GenericSrv.h"
+#include "rclcUtilities.h"
+#include "ue_msgs/srv/spawn_entities.h"
+
+#include "ROS2SpawnEntitiesSrv.generated.h"
+
+// potential problem: if this struct is defined multiple times!
+USTRUCT(Blueprintable)
+struct RCLUE_API FROSSpawnEntities_Request
+{
+	GENERATED_BODY()
+
+public:
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> type;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> name;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> position;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FQuat> orientation;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> twist_linear;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> twist_angular;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> reference_frame;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> tags;
+
+	
+
+	void SetFromROS2(const ue_msgs__srv__SpawnEntities_Request& in_ros_data)
+	{
+    	for (int i = 0; i < in_ros_data.type.size; i++)
```

Comment:
```suggestion
    	for (auto i = 0; i < in_ros_data.type.size; ++i)
```
as configured [here](https://github.com/rapyuta-robotics/UE_tools/blob/devel/CodeGen/CodeGeneratorFromPath.py#L151)?